pkg/helmexec/exec_test.go

```diff
@@ -752,18 +753,6 @@ func Test_ChartExport(t *testing.T) {
 			chartFlags:  []string{"--untar", "--untardir", "/tmp/dir"},
 			listResult: `Exporting chart
 exec: helm --kube-context dev chart export chart --destination path1 --untar --untardir /tmp/dir
```

Comment:
why remove this test case?

Reply:
Apparently, this has been removed as unnecessary. With the changes made in this PR, the target function `helm.ChartExport` (See L771) is no longer called for helm 3.7 or greater. We have two functions `ChartPull` and `ChartExport` and it's `ChartPull` which Helmfile calls for helm 3.7+, and that's already tested in the existing `Test_ChartPull`.